README.md

```diff
@@ -250,4 +250,57 @@ POST
     - if course is added successfully
         - `{"success": true, "message": "New Course successfully saved."}`
     - if course could not be added
-        - `{"success": false, "msg": "Unexpected error."}`
\ No newline at end of file
+        - `{"success": false, "msg": "Unexpected error."}`
+
+**/updateTeams**  
+
+POST
+- When the user is logged in and selects 'save teams' invoke gets data from new applicant form and passes into insertNewApplicantToDb function for inserting into database.
+- Returns success true / false
+  - If successful it returns 'success true' with message 'Team successfully saved' 
+  - If trainer name is missing it returns success => false with message 'Missing trainer names'
```

Comment:
Look at the other endpoint documentation examples: there are little example JSON snippets for both sends and returns. You need that here as well.